src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString.length === 0) {
+    return {};
+  }
+
+  const GROUP_STYLE = sourceString.trim().split(';');
+  const GET_CLEAN_STYLE = GROUP_STYLE.map((pair) => pair.trim()).filter(
+    (style) => style !== '',
+  );
+
+  const SEPARATION_STYLE = GET_CLEAN_STYLE.map((pair) => {
+    return pair.trim().split(':');
+  });
```

Comment:
The variable 'SEPARATION_STYLE' could be named more descriptively to reflect its contents, such as 'stylePairs'. Additionally, you don't need to call trim() here, as it was already called on each pair in the previous map function.